### ISSUE TITLE
Dropin crashes App NSInvalidArgumentException

### ISSUE THREAD

**Issue by bennibau** [braintree/braintree-ios-drop-in#331]
**Integration Details (please complete the following information):**
 - SDK/Library Version: 9.0.1
 - Environment: Sandbox
 - iOS Version:  iOS 14.5
 - Device iPhone 12 mini
 - Integration type: CocoaPods
 
The SDK is used in a Flutter App but that does not matter for this bug.

**Describe the bug**
When opening dropin the app crashes.
Debugging showed that the SDK is trying to read NSString on a BTJson object when comparing the creditCards in the configuration with Strings.

in the file `BTDropinController` line 227:

```
for (NSString *supportedCardType in self.configuration.supportedCardTypes) {
                    BTDropInPaymentMethodType paymentMethodType = [BTUIKViewUtil paymentMethodTypeForPaymentInfoType:supportedCardType];
                    if (paymentMethodType != BTDropInPaymentMethodTypeUnknown) {
                        [paymentMethodTypes addObject: @(paymentMethodType)];
                    }
               }
```

and in `BTPaymentSelectionViewController` line 137

```
for (NSString *supportedCardType in self.configuration.supportedCardTypes) {
                BTDropInPaymentMethodType paymentMethodType = [BTUIKViewUtil paymentMethodTypeForPaymentInfoType:supportedCardType];
                if ([BTUIKViewUtil isPaymentMethodTypeACreditCard:paymentMethodType] && !self.dropInRequest.cardDisabled) {
                    // Add credit cards if they are supported
                    [activePaymentOptions addObject:@(BTDropInPaymentMethodTypeUnknown)];
                    break;
                }
            }
```

**To Reproduce**
1. Open Dropin
2. Crash

**Expected behavior**
No crash expected

This PR is probably fixing it:

https://github.com/braintree/braintree-ios-drop-in/pull/329

Please merge asap.

**Comment by scannillo** [comment]
Hi @bennibau - thank you for opening this issue. We will get a fix out for this ASAP.

In the meantime, you can specify that DropIn v9.0.1 uses Braintree v5.3.1 in your Podfile:

```
pod 'Braintree', '5.3.1'
```

**Comment by sestevens** [comment]
@bennibau, we just released a fix for this in [version 9.0.2](https://github.com/braintree/braintree-ios-drop-in/releases/tag/9.0.2).
